miqi/runtime/task_runner.py

```
@@ -600,6 +600,51 @@ async def _handle_user_message(self, msg: UserMessage) -> None:
+        # agent_registry 主提示词规则 7 要求 agent 先查 "Local Skills" 列表，
+        # 但该列表此前从未注入，导致 agent 唯一的发现途径是 skill_manage(list)
+        # （量化评估: 14 条直接提示词 11 条零技能接触）。这里按渐进披露注入
+        # Layer 1（名称+描述+位置），技能正文仍由 agent 按需加载。
+        if _ws is not None:
+            try:
+                from miqi.agent.skills import SkillsLoader
+
+                _skills_summary = SkillsLoader(_ws).build_skills_summary(
+                    description_max_chars=160,
+                )
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)